Tutorials/NetKet_Tools/custom_graph.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2018 The Simons Foundation, Inc. - All Rights Reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#    http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Demonstrates the use of the pynetket to calculate ground state of simple
+Ising1D model.
```

Comment:
Ising model on a star graph in this case

Reply:
Sure thing! I also updated the name in the description to netket_tools.